### DISCUSSION TITLE
Error trying to train a model in fast-dreambooth on paperspace :((

### DISCUSSION THREAD

**Discussion by choriflai** [TheLastBen/fast-stable-diffusion#2144] [Q&A] [answered]
Hi, I've paid the service in paperspace due to recent Google Colab restrictions. I loved this notebook since colab but now that I tried to run it on paperspace it throws me an error of "CUDA out of memory" trying to train the text encoder (When the error shows up it skips the text encoder step and start with Unet, so the results are none like the instance images".
This notebook on gradient its working for everyone else?
![Error1](https://github.com/TheLastBen/fast-stable-diffusion/assets/112909734/ea515cb2-cbc0-4970-ae66-03a25cf45502)
Have a great week!. (sorry if there's some errors with my english, it's not my language :c )

**Comment by TheLastBen** [comment]
at what resolution are you training ?

**Reply by choriflai** [reply]
I'm training at 1024x1024, but tried to reduce it to 768x768 and 640x640 and keep getting the same error (on rtx5000 gpu with 16gb, I don't know if its enough)
What's strange it's that I trained one model at 1024x1024 on google colab with a Tesla T4 so I don't think that have much to do with that

**Reply by TheLastBen** [reply]
rerun the first cell (takes 3 seconds) then try running the training again

**Reply by choriflai** [reply]
Thanks, I'll try it now and tell how it goes

**Reply by choriflai** [reply]
You mean without uploading instance images?
Just dependencies, model and session, right?

**Reply by choriflai** [reply]
Well, I'm getting the same. I don't know if I'm doing something wrong. You have it running in paperspace lately? 
It throws the error and keeps with the next step that's training the Unet. I can keep it running until it finishes and saves a ckpt file, but the results are not the instance image 'cause it has no text encoder I assume :c

**Reply by choriflai** [reply]
![error2](https://github.com/TheLastBen/fast-stable-diffusion/assets/112909734/b9eedd18-5767-453f-95e0-06c751cfee3b)

**Reply by TheLastBen** [reply, marked as answer]
you can't use dreambooth with an 8GB GPU

**Reply by choriflai** [reply]
Oh, I didn't notice that I picked a smaller one now, mb
The notebook it's working for you?
To see if I'm doing it wrong or its just an error in the notebook
Works for me in colab, but I'm having this trouble in paperspace 
Thanks for responding btw I'm a newbie in all this

**Reply by TheLastBen** [reply]
if you're encountering errors, copy the log here

**Reply by choriflai** [reply]
I ran the latest notebook on /PPS and on a more powerful gpu (a4000), and seems to be working. I think that the first problem was that I was running an outdated notebook, and then the gpu with low memory incident
Now on the A4000 it's not showing any errors so that was it. Thanks for the help, just mb!

**Comment by cutecoelho** [comment]
Why you don't use the Google Colab to train models? Still working on the free plan. Only A1111 isn't working.

**Reply by TheLastBen** [reply]
it's inconvenient if you can't test the trained model with the webui

**Reply by choriflai** [reply]
Yeah, I can't test the results anymore, and because of Colab problem running automatic1111 I paid paperspace anyways, so in theory I have access to much powerful gpus (Training of 3000 steps on colab was like 2:30hrs on a T4, and now if I can catch an rtx A4000 it's like 15-20min)

**Reply by cutecoelho** [reply]
Ohh, get it. I never used paperspace, don't know how it works. Thought would be a easier way to upload the model and test it
